ndk-glue/Cargo.toml

```diff
@@ -18,7 +18,7 @@ ndk-macro = { path = "../ndk-macro", version = "0.2.0" }
 lazy_static = "1.4.0"
 libc = "0.2.66"
 log = "0.4.8"
-android_logger = { version = "0.8.6", optional = true }
+android_logger = { version = "0.9.2", optional = true }
```

Comment:
Is there anything particular needed from `.2` that is not available in `.0`? Otherwise I prefer to write just `"0.9"` here though the rest of the file doesn't adhere to that either. What do the other maintainers think?

Reply:
Switched to `0.9`.